src/main/java/core/basesyntax/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class RegistrationException extends NullPointerException {

```

Comment:
Using NullPointerException as a base for RegistrationException is not a good practice, since NullPointerException is a runtime exception that usually indicates a programming error, such as dereferencing a null object. Instead, create a custom exception that extends RuntimeException or Exception directly, depending on whether you want it to be a checked or unchecked exception.

Reply:
Done